src/util/variantUtils.ts

```diff
@@ -38,5 +40,5 @@ export function variantToMutation(
 
 export function getProteinPosStart(proteinChange: string | undefined) {
     const proteinPosition = getProteinPositionFromProteinChange(proteinChange);
-    return proteinPosition ? proteinPosition.start : 0;
+    return proteinPosition ? proteinPosition.start : -1;
```

Comment:
set to `-1`, otherwise when there is no protein position it will show lollipop at `0` postion

Reply:
i guess this was correct for TERT promotor: https://deploy-preview-89--genome-nexus-frontend.netlify.app/variant/5:g.1295228G%3EA

Reply:
not sure exactly how we would separate that case tho

Reply:
maybe based on `c.x` position or something

Reply:
maybe ok to leave as -1 for now